### PR TITLE
Test suite updates to fix ssh pubkey access to the tested VM

### PR DESCRIPTION
**Pull request by matusmarhefka**
#### Description:

- Fixed the code exporting `SSH_ADDITIONAL_OPTIONS` which was not working correctly and exported only one ssh option omitting others.
- `unselect_rules_list` file updated with rules which might break the ssh access to VM.

### CODE REVIEW

File: tests/unselect_rules_list [hunk 1]
```diff
@@ -3,23 +3,11 @@ xccdf_org.ssgproject.content_rule_ensure_gpgcheck_local_packages
...
-xccdf_org.ssgproject.content_rule_grub2_uefi_password
-xccdf_org.ssgproject.content_rule_installed_OS_is_certified
-xccdf_org.ssgproject.content_rule_install_antivirus
-xccdf_org.ssgproject.content_rule_install_hids
-xccdf_org.ssgproject.content_rule_install_mcafee_antivirus
-xccdf_org.ssgproject.content_rule_install_mcafee_cma_rt
-xccdf_org.ssgproject.content_rule_install_mcafee_hbss_accm
-xccdf_org.ssgproject.content_rule_install_mcafee_hbss_hips
-xccdf_org.ssgproject.content_rule_install_mcafee_hbss_pa
-xccdf_org.ssgproject.content_rule_mcafee_antivirus_definitions_updated
```
Comment (yuumasato) [inline tests/unselect_rules_list]: @matusmarhefka Was the removal os this rules expected as well? They are not mentined in PR description.

Comment (matusmarhefka) [reply tests/unselect_rules_list]: @yuumasato Yes it was, these rules have no remediation, therefore they can remain selected.

Comment (yuumasato) [reply tests/unselect_rules_list]: @matusmarhefka The rules at the top of the chunk have remediation, like the ones related to `rule_rpm_verify_hashes`, permissions and ownership. Should they be removed as well?

Comment (matusmarhefka) [reply tests/unselect_rules_list]: Yes, there should be only rules which might break the access to the machine for the test suite or destructive rules (which influence other rules remediations). We want to test rpm_verify family of rules. Of course, we can update this list in the future, so if you think that some rules should be added/removed from the list, please open a PR and we can discuss there.

Comment (yuumasato) [reply tests/unselect_rules_list]: Just wanted to be sure. :)

**Pull request merged**